src/views/Claims.vue

```diff
@@ -98,10 +119,21 @@ export default class Claims extends Vue {
             from: this.account,
         });
     }
+    async insetWallet() {
+        try {
+            await axios.post(API_URL_POST_WALLET, {
```

Comment:
Same as the other comment.

It's better to leave the section that is mostly static in. The reason for this is that this is position specific as well.
The URL prefix isn't, that is always the same for the API. Could do something like: `${VUE_APP_API_URL}\wallet\${this.account}`
(Above example is generic and not specific to this line)

Storing all possible routes in the .env. creates a lot of clutter.